app/controllers/application_controller.rb

```diff
@@ -1,2 +1,19 @@
 class ApplicationController < ActionController::Base
+  before_action :authorize
+
+  private def redirect_to_homepage_if_logged_in
+    return unless session[:user_id]
+
+    redirect_to store_index_path, notice: t('notice.application.logged_in')
```

Comment:
user root path here, and update that in routes file if required.

Reply:
Done